feature/tabsflow/moreflow/faq/src/main/java/com/epmedu/animeal/tabs/more/faq/presentation/FAQScreenUI.kt

```diff
@@ -0,0 +1,91 @@
+package com.epmedu.animeal.tabs.more.faq.presentation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.foundation.preview.AnimealPreview
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.resources.R
+import com.epmedu.animeal.tabs.more.faq.presentation.FAQScreenEvent.BackClicked
+import com.epmedu.animeal.tabs.more.faq.presentation.FAQScreenEvent.CardClicked
+import com.epmedu.animeal.tabs.more.faq.presentation.model.FAQCard
+import com.epmedu.animeal.tabs.more.faq.presentation.ui.FAQHeader
+import com.epmedu.animeal.tabs.more.faq.presentation.ui.toComposable
+import com.epmedu.animeal.tabs.more.faq.presentation.viewmodel.FAQState
+import com.epmedu.animeal.tabs.more.faq.util.generateLoremIpsum
+import kotlinx.collections.immutable.toImmutableList
+
+@Composable
+internal fun FAQScreenUI(
+    state: FAQState,
+    onEvent: (FAQScreenEvent) -> Unit
+) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .statusBarsPadding(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = remember {
+            // Sticks last element to the bottom
+            object : Arrangement.Vertical {
```

Comment:
May be refactor anonymous inner class to separate class?

Reply:
Done